### PR TITLE
Add howto documentation on issue tracker usage.

### PR DESCRIPTION
**Pull request by MaineC**
The way InnerSource projects use issue trackers differs from how traditional
agile teams are using this technology. In order to avoid confusion because of
that make the different use cases for issue trackers in InnerSource projects
explicit.

### CODE REVIEW

**Comment by MaineC** [comment]
Relates to #114

**Review by rrrutledge — APPROVED** [review]
Great!  I have a few points about what parts of context should be added to a backlog item that I'd like to add here after this is merged.

**Comment by MaineC** [comment]
Just for the sake of transparency: While this is approved, I cannot merge it myself. (I'm not asking for commit access, just trying to avoid others waiting for me to click the merge button ;) )

**Comment by rrrutledge** [comment]
Thanks for this call-out, Isabel!  I asked in the [#innersource-patterns](https://innersourcecommons.slack.com/archives/C2EFRTS6A/p1578407461000400) channel about merging it.

**Comment by spier** [comment]
To follow the Pattern Template format more closely, I would suggest to make two additions:
* add Patlet
* add Status (I assume **Unproven**?

Other than that, LGTM.

**Comment by MaineC** [comment]
@spier You're right, this pattern would benefit from these two sections. Thanks for pointing that out. Fixed now.

**Comment by spier** [comment]
LGTM @lenucksi.

What I don't know is where in the README this pattern should be added to?

**Review by lenucksi — APPROVED** [review]
Thanks for this great pattern @MaineC :100:
I've seen the dreaded _"title-only" issue trackers_ a few times too often. 
I'm going to merge this now so that @rrrutledge can add his additions. 
@MaineC: Could you please suggest a position within the readme.md file for this pattern by means of a second pull request mentioning this one?

**Pull request merged**
